Sources/Hyperspace/Request/Decoding/DecodableContainer.swift

```diff
@@ -31,9 +31,9 @@ extension AnyRequest where T: Decodable {
                                        timeout: TimeInterval = RequestDefaults.defaultTimeout,
                                        decoder: JSONDecoder = JSONDecoder(),
                                        containerType: U.Type) where U.ContainedType == T {
-        self.init(method: method, url: url, headers: headers, body: body, cachePolicy: cachePolicy, timeout: timeout) { data in
+        self.init(method: method, url: url, headers: headers, body: body, cachePolicy: cachePolicy, timeout: timeout) { success in
             do {
-                return try .success(decoder.decode(U.ContainedType.self, from: data, with: U.self))
+                return try .success(decoder.decode(U.ContainedType.self, from: success.data, with: U.self))
             } catch {
                 return .failure(AnyError(error))
```

Comment:
Total side note, but I think now with the introduction of `DecodingFailureInitializable` this can be condensed into `Request`. I'll take a look at doing that, I have some other work I'd like to do with error handling.